docs/data/destinations/braze.md

```diff
@@ -60,12 +60,14 @@ Under **Send Events**, make sure the toggle is enabled ("Events are sent to Braz
 2. (optional) In **Select additional properties**, select any more event and user properties you want to send to Braze. If you don't select any properties here, Amplitude doesn't send any. These properties are sent to Braze as [Braze custom event properties](https://www.braze.com/docs/user_guide/data_and_analytics/custom_data/custom_events/#custom-event-properties). _Transformed event properties and transformed user properties aren't supported._
 
 ### Configure user forwarding
+!!!warning "Temporarily Disabled"
+    We're actively working on improving user forwarding to address concerns around high call volumes and to provide more functionality to customers. All existing active user forwarding syncs will continue to operate, but the ability to enable it on any sync configurations where it's not already enabled has been temporarily removed.
```

Comment:
```suggestion
    We're actively working on improving the experience of user forwarding to Braze, including addressing concerns around high call volumes which can incur additional costs in Braze. All enabled user forwarding syncs will continue to run. Temporarily, no new user forwarding syncs can be enabled. If an enabled user forwarding sync is disabled, it can not be re-enabled at this time. Event forwarding to Braze is unaffected by this and can still be configured as normal. User forwarding destinations other than Braze are also unaffected and can still be configured as normal.
```